platforms/cortex/trace_utils.c

```diff
@@ -0,0 +1,62 @@
+/* System includes */
+#include <stdlib.h>
+
+/* RIOT includes */
+#include "xtimer.h"
+
+/* Application includes */
+#include "trace_utils.h"
+#include "tracefd.h"
+#include "platform.h"
+#include "obstacles.h"
+
+/* Trace filename */
+#define TRACE_FILE "trace.txt"
+
+/* Trace file descriptor on sdcard */
+static tracefd_t tracefd_sdcard;
+
+/* Periodic task */
+#define TASK_PERIOD_MS 100
```

Comment:
what is the flush interval then ?

Reply:
The state is printed every 100ms while the flush is performed every 1s.
It should be 60ms here, will fix that.